style.css

```diff
@@ -15,10 +15,106 @@ body {
   color: #555;
 }
 
+.blur-add {
+  filter: blur(8px);
+  -webkit-filter: blur(8px);
+}
+
+.blur-remove {
+  filter: none;
+  -webkit-filter: none;
+}
+
 .header {
   display: none;
 }
 
+.mobile-popup {
```

Comment:
- Please make sure the popup follows the design for both desktop and mobile versions from the chosen template.
- Your design: It seems fine but not as you see it because I checked from different computers. 
- Hint1:  add "bottom" to this class. (mobile-popup)
- Hint2: Go to class ".pop-bottom" and try to work on "margin" there. you can also try to take it(margin) off

![ok](https://user-images.githubusercontent.com/63972670/167892989-36594d67-e78e-4201-aa39-92d7814de91b.PNG)